.idea/encodings.xml

```diff
@@ -1,4 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why are we removing the directory? I am not sure if our design team needs it for updates, for example.

Does it break anything at the site?

Reply:
I removed that since the dir is not tracked by git anyway (it is in `.gitignore`). Maybe we can ask somebody in the design team. Can you point me to someone?

Reply:
Ah, sorry. Wasn't aware we have it at `.gitignore`.

Looks it was added in advance by mistake, and never removed, but just in case... Ping @en3sis, can we safely remove the `.idea` directory or you need it for any kind of design changes in the future?

Reply:
Hello guys,
Yes, it can be removed (it has something to do with IDE preferences and nothing with the project)

Reply:
Thanks a lot for confirming @en3sis :-)

@hustodemon merging the PR then. Thanks for the contribution.

Reply:
Thanks guys!